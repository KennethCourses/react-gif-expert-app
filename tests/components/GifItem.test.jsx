import { render } from '@testing-library/react'
import GifItem from '../../src/components/GifItem';

describe('Testing to GifItem component.', () => {

    const title = 'Kenneth';
    const url = 'https://localhost/image.jpg';

    test('Should show the component correctly', () => {

        const { container } = render(
            <GifItem
                title={title}
                url={url}
            />
        )

        expect(container).toMatchSnapshot();
    })
})