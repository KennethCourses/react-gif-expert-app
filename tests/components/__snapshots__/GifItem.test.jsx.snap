// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing to GifItem component. Should show the component correctly 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="Kenneth"
      src="https://localhost/image.jpg"
    />
    <p>
      Kenneth
    </p>
  </div>
</div>
`;
